src/GenFacades.Core.Desktop/GenFacades.Core.Desktop.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
Oh okay, I misunderstood you. I will try to see if we can just have one single GenFacades.Core library and reference it from both. I think it should be possible, since we are doing that for Cci.Extensions which has roughly the same dependencies.